src/SALib/analyze/morris.py

```diff
@@ -178,31 +189,39 @@ def _compute_statistical_outputs(
         Array defining the distribution of groups
     unique_group_names: List
         Names of the groups
+     scaled_elementary_effects: np.ndarray
+       the mean of the absolute elementary effect scaled by the ratio of the standard deviation of the input over the output
 
     Returns
     -------
     Si: ResultDict
         Morris statistical parameters.
     """
 
-    Si = ResultDict(
-        (k, [None] * num_vars)
-        for k in ["names", "mu", "mu_star", "sigma", "mu_star_conf", "scaled_EE"]
-    )
-
     mu = np.average(elementary_effects, 1)
     mu_star = np.average(np.abs(elementary_effects), 1)
     sigma = np.std(elementary_effects, axis=1, ddof=1)
     mu_star_conf = _compute_mu_star_confidence(
         elementary_effects, num_vars, num_resamples, conf_level)
-    scaled_EE = np.average(np.abs(see_trajectory), 1)
+    
+    if scaled_elementary_effects is None:
```

Comment:
This if-statement logic is not required as scaled elementary effects should always be produced.